docs/application/native/guides/device/battery-monitor.md

```diff
@@ -79,90 +77,105 @@ To get the battery usage information related to the application ID:
     }
     ```
 
-- Get total battery usage of an application ID over a specific duration of time.
-  To get a particular time duration type, use the `battery_monitor_duration_type_e` enum:
+- Get the total battery usage of an application ID over a specific interval of time. The timestamps **from** and **to** must be set as per the Unix Epoch time format:
 
     ```
-    duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
-    error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+    /* To get the usage for past 24 hours from the current epoch time */
+    int period_t = 24*60*60; //24 hours in seconds
+    to = now; from = to - period_t;
+
+    error_code = battery_monitor_get_power_usage_by_app(app_id, from, to, &battery_usage);
     if (error_code == BATTERY_MONITOR_ERROR_NONE)
-        printf("The total battery usage for appid [%s] over last 24 hrs is [%d]", app_id, battery_usage);
+        printf("The total battery usage for appid [%s] over last 24 hrs is [%lf]", app_id, battery_usage);
+    else if (error_code == BATTERY_MONITOR_ERROR_RECORD_NOT_FOUND)
+        printf("The Battery Usage for appid [%s] is not recorded");
     else
         printf("Error Occurred [%d]", error_code);
 
-    duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
-    error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+    /* To get usage for a period of thousand seconds starting, 2 hours before the current epoch time */
```

Comment:
Question.
According to the sample code, it is said

to = current epoch time - 2 hours
from = current epoch time - 2 hours - 1000 seconds

So it seems to calculate the usage for a period of 1000 seconds between "before 2 hours and 1000 seconds" and "before 2 hours", according to the code.

But this sentence seems to calculate the usage for a period of 1000 seconds starting 2 hours before, which means between "before 2 hours" and "after 1000 seconds from before 2 hours"

Maybe I confused something, but would you please check the sentence again and clear my doubt?


| 2 hours and 1000 seconds before | 2 hours before| 1000 seconds after the before 2 hours| current|
|-|-|-|-|
|a|b|c||

Which one is correct? a-b or b-c?